packages/block-renderer/src/components/block-renderer-container.tsx

```diff
@@ -19,7 +19,7 @@ interface BlockRendererContainerProps {
 	inlineCss?: string;
 	viewportWidth?: number;
 	viewportHeight?: number;
-	maxHeight?: string | number;
```

Comment:
Consumers of the component only pass numbers or "none", and the component implementation only handles none and number types. (E.g. its logic doesn't work with a value like "100px") So the string type wasn't specific enough which caused some issues in the component implementation.